src/main/resources/db/changelog/changes/v1.0.0/sql/2021-08-02-01__create-patron-type-mapping-table.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE patron_type_mapping
+(
+    id UUID NOT NULL,
+    patron_group_id UUID NOT NULL UNIQUE,
```

Comment:
Folio patron group should be unique within central server. Same patron group can be mapped multiple times, for different server. 
Also please define unique constraint separately from field specification